etc/systemd/system/.gitignore

```diff
@@ -1,3 +1,4 @@
 *.service
 *.target
 *.preset
+*.timer
```

Comment:
Is it necessary for the .timer files to be .timer.in? I'm not seeing any substitutions on them. Does it work if you just have them named .timer in the source tree (and then remove this .gitignore, of course)?

Reply:
```make
make[4]: *** No rule to make target 'zfs-scrub-monthly@.timer.in', needed by 'zfs-scrub-monthly@.timer'.  Stop.
make[4]: *** Waiting for unfinished jobs....
```

```make
SUBSTFILES += $(systemdpreset_DATA) $(systemdunit_DATA)
```
so it still runs a noop substitution on the files.

to keep it simple I did not want to touch to much `Makefile.am` and introduce extra filelist with static files and merge it later to _data after